src/types/entities/player.ts

```diff
@@ -60,7 +73,7 @@ export class Player {
 	 */
 	format(): string {
 		let result = this.overlayString()
-
+		if (this._isPlaying) result = '{blue-bg}' + result + '{/blue-bg}'
```

Comment:
In this context, we could pull something like `game.isPlaying(overlay, round)`, where we define an `isPlaying` public method on `SchellingGame` that returns `rounds.get(round).players.includes(overlay)`.

Reply:
See my comment above about the cross-coupling out of Player back to the game.  And the Player doesn't know what round to query because it really wants to know if it is playing in the "current" round which only the game should understand.   So your proposed code would actually be more like `rounds.get(this.currentRoundNo).players.includes(overlay)` which is rather performance intensive every time the Player renders compared to having the Player already know if it should be highlighted or not, which really doesn't change that often.

I'm a firm believer in implementing performance where possible rather than needing to go back and figure out where it needs to be done after a system is already in trouble.